src/github.com/getlantern/flashlight/settings/settings.go

```diff
@@ -39,3 +61,20 @@ func start(baseSettings *Settings) error {
 	service, err = ui.Register(messageType, nil, helloFn)
 	return err
 }
+
+func read() {
+	for msg := range service.In {
+		settings := (msg).(map[string]interface{})
+		config.Update(func(updated *config.Config) error {
+
+			autoReport := settings["autoReport"].(bool)
+			if autoReport {
+				analytics.Start()
+			}
+
+			baseSettings.AutoReport = autoReport
+			*updated.AutoReport = autoReport
+			return nil
+		})
```

Comment:
Do we also need to publish the updated settings to other clients?  Maybe that's actually best done in Configure()?